### PR TITLE
Added unit tests for better code coverage (#368)

### PR DESCRIPTION
**Pull request by dilraj45**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `master` branch.
- [x] I have added necessary documentation (if appropriate)

#### Changes proposed in this pull request:

- Unit test for `server.py`

Screenshot of current coverage report
coverage

### CODE REVIEW

**Comment by Remorax** [comment]
Please resolve conflicts 😄

File: .travis.yml [hunk 1]
```diff
@@ -10,8 +10,8 @@ install:
 before_script:
   - flake8 . --count --max-complexity=15 --show-source --statistics
 script:
-  - python app/server.py > /dev/null &
-  - pytest --capture=sys --showlocals
```
Comment (cclauss) [inline .travis.yml]: Why remove __—showlocals__?  I find that option quite helpful.

Comment (dilraj45) [reply .travis.yml]: I will add the `showlocals` option. I guess, I overlooked this while fixing conflicts, apologies for that

**Comment by ParthS007** [comment]
@bhaveshAn @vaibhavsingh97  Please review

**Pull request merged**